app/controllers/ui_job_wizard_controller.rb

```diff
@@ -8,6 +8,14 @@ def categories
     render :json => {:job_categories =>job_categories}
   end
 
+  def template
+    job_template = JobTemplate.find_by(id: params[:id])
+    render :json => {
+      :job_template => job_template,
+      :effective_user => job_template.effective_user,
```

Comment:
Is it necessary to extract the effective user out of a job template? In other words, can't the frontend code just take it from the template?

Reply:
It's not in a the template because its a function I think:
https://github.com/theforeman/foreman_remote_execution/blob/f7e711f6aa54300497f967a6d90576133ab87863/app/models/job_template.rb#L131

Reply:
Nevermind then